dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -524,52 +523,47 @@ public String builds(@ApiParam(hidden = true) @Auth Token authToken, @QueryParam
         return cwl;
     }
 
-    private static class QuayUser {
-        private String username;
-
-        public void setUsername(String username) {
-            this.username = username;
-        }
-
-        public String getUsername() {
-            return this.username;
-        }
-    }
-
     @GET
     @Timed
     @UnitOfWork
     @Path("/getQuayUser")
-    @ApiOperation(value = "Get quay user", notes = "testing", response = QuayUser.class, hidden = true)
-    // , hidden = true)
-    public QuayUser getQuayUser(@ApiParam(hidden = true) @Auth Token authToken, @QueryParam("tokenId") long tokenId) {
+    @ApiOperation(value = "Get quay user", notes = "testing", response = ArrayList.class, responseContainer = "List", hidden = true)
```

Comment:
Can you expand the notes and value here? 
It isn't clear to me what is being returned here. Try creating your own return type here so that Swagger has more to go on here than a list of lists?